# バッテリ電圧GET
---
float64 value

